transform/mattermost-analytics/models/marts/product/licenses/_product__licenses__models.yml

```diff
@@ -0,0 +1,90 @@
+version: 2
+
+models:
+  - name: dim_license_customers
+    description: |
+      List of known customers with active on prem licenses. Note that the same customer might appear more than once in
```

Comment:
since these are on-prem/self-hosted only, maybe it make sense to make the distinction in the name/folder structure somehow.